src/styles/calendar.scss

```
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: get-width($ceil-width, $gap);
+  padding: 0;
+
+  &__item {
+    position: relative;
+    box-sizing: border-box;
+    height: $ceil-height;
+    width: $ceil-width;
+    background-color: #eee;
+    list-style: none;
+    transition: 0.5s ease;
```

it is always better to be specific about transform
```suggestion
     transition: background-color 0.5s, transform 0.5s;
```